### ISSUE TITLE
onLoadMore does not get called anymore

### ISSUE THREAD

**Issue by abhiakshay** [CassetteRocks/react-infinite-scroller#239]
upon scroll to the end, does not trigger the onLoadMore despite hasMore=true
here is the configuration used
threshold={200}
initialLoad={false}
loadMore={() => console.log('load more')}
hasMore={true}
useWindow={false}

**Comment by brianwilliams-mubaloo** [comment]
This has been driving me nutty. I'm glad it's not just me. Is there an ETA on this?

**Comment by DarthCharles** [comment]
If you do it slowly it loads more rows, but if you scroll fast enough to the end nothing gets called, is that what you're seeing as well @brianwilliams-mubaloo @abhiakshay  ?

**Comment by grazianodev** [comment]
@brianwilliams-mubaloo @abhiakshay Did you find a solution? It suddenly stopped working for me too...

**Comment by lkavetskiy** [comment]
It doesn't work for me too.

**Comment by lubomirKavetskiy** [comment]
It doesn't work for me too.

**Comment by dcporter44** [comment]
i just started trying to use this library, and when I scroll to the bottom it just repeatedly calls `loadMore` until it's out of pages. not sure if this is the same thing you're experiencing

**Comment by grazianodev** [comment]
Nope, but I had that issue as well and I solved it by setting `initialLoad={ false }` and fetching the first page manually, which was a requirement for me anyway. When you do that, the library fetches the subsequent pages correctly. 

I also seem to have fixed the issue discussed in this thread. For me, what works (at least for me) is adding a 10px  padding at the bottom of a container div that wraps the loading component.

**Comment by alex-shamshurin** [comment]
@grazianodev Yes, it solves the issue

**Comment by damechen** [comment]
Hit the same issue. Driving me nuts. Any update on this issue?

**Comment by damechen** [comment]
@grazianodev It didn't work for me :(

**Comment by stanojevic-ivan-toptal** [comment]
> i just started trying to use this library, and when I scroll to the bottom it just repeatedly calls `loadMore` until it's out of pages. not sure if this is the same thing you're experiencing

What you can do as work around is set flag like isLoading, when you send your request set it to true, on response false, and set hasMore=(!isLoading && (...)).

**Comment by NguyenTrungMax** [comment]
it didn't work for me,

**Comment by divjyot3112** [comment]
loadmore is not getting called!

**Comment by mqliutie** [comment]
```jsx
<div style={{ height: document.body.clientHeight - 250, overflow: 'auto' }}>
    <InfiniteScroll
        initialLoad={false}
        loadMore={async (current) => {
            console.log('loadmore will never trigger', current);
            this.setState({requesting:true})
            await myRequest();
            this.setState({requesting:false})
        }}
        hasMore={!this.state.requesting && this.state.hasNext}
        threshold={20}
        useWindow={false}
    >
        <Table />
    </InfiniteScroll>
</div>
```

**Comment by jamessspanggg** [comment]
> Nope, but I had that issue as well and I solved it by setting `initialLoad={ false }` and fetching the first page manually, which was a requirement for me anyway. When you do that, the library fetches the subsequent pages correctly.
> 
> I also seem to have fixed the issue discussed in this thread. For me, what works (at least for me) is adding a 10px padding at the bottom of a container div that wraps the loading component.

experiencing this as well and the above solution works for me, but is there an explanation to this?

**Comment by atinyakov** [comment]
In my case I forgot to add `height `attribute to the parent `<div>`. Once added the issue got resolved.

**Comment by Emad-salah** [comment]
I figured out the cause of this issue in my case, I was trying to use the infinite scroll functionality with `useWindow={true}` and the issue turned out to be caused by a parent div which had `overflow: auto` specified, I got rid of that and everything works now as expected! 😄

**Comment by olelivalife** [comment]
> `w: auto` specified, I got rid of that and everythi

That's weird 'cause [the examples](https://www.npmjs.com/package/react-infinite-scroller) uses parent with `overflow: auto;`

**Comment by Dylan-Kerler** [comment]
loadMore is not working for me

**Comment by arvidnilber** [comment]
Any update on this issue? This randomly stopped working for me. Works if I remove initialLoad={false} but then the function calls infinitely times.

**Comment by lazycipher** [comment]
Is this project dropped or something?
I tried the above-mentioned fixes but sadly none worked.
